chat-plugins/faq.js

```diff
@@ -0,0 +1,72 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+
+const ROOMFAQ_FILE = path.resolve(__dirname, '../config/chat-plugins/faqs.json');
+const ALLOWED_HTML = ['a', 'font', 'i', 'u', 'b'];
+
+let roomFaqs = {};
+try {
+	roomFaqs = require(ROOMFAQ_FILE);
+} catch (e) {
+	if (e.code !== 'MODULE_NOT_FOUND') throw e;
+}
+if (!roomFaqs || typeof roomFaqs !== 'object') roomFaqs = {};
+
+function saveRoomFaqs() {
+	fs.writeFileSync(ROOMFAQ_FILE, JSON.stringify(roomFaqs));
+}
+
+exports.commands = {
+	addfaq: function (target, room, user) {
+		if (!this.canTalk()) return this.errorReply("You cannot do this while unable to talk.");
+		if (!this.can('declare', null, room)) return false;
+		if (!room.chatRoomData) return this.errorReply("This command is unavailable in temporary rooms.");
+		let [topic, ...text] = target.split(',');
```

Comment:
I didn't really see the point for (small) FAQ messages. Was thinking about adding it later if people really wanted it.